presto-kafka/src/main/java/io/prestosql/plugin/kafka/schemareader/SchemaReader.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka.schemareader;
+
+import java.util.Optional;
+
+public interface SchemaReader
+{
+    SchemaReader DUMMY_SCHEMA_READER = (topicName, dataSchemaLocation) -> Optional.empty();
+
+    Optional<String> readSchema(String topicName, Optional<String> dataSchemaLocation);
```

Comment:
One issue is that schema may refer to different things schema in Presto and schema definition of content in Kafka. I think we need to have two names for that otherwise it is misleading. 

How about `KafkaSchema` or `ContentSchema`?